src/templates/validate.yaml

```diff
@@ -2,20 +2,19 @@ apiVersion: admissionregistration.k8s.io/v1beta1
 kind: ValidatingWebhookConfiguration
 metadata:
   annotations:
-    cert-manager.io/inject-ca-from: seldon-system/seldon-serving-cert
+    cert-manager.io/inject-ca-from: kubeflow/seldon-serving-cert
   creationTimestamp: null
   labels:
     app: seldon
-    app.kubernetes.io/instance: seldon1
-    app.kubernetes.io/name: seldon
-    app.kubernetes.io/version: v0.5
-  name: seldon-validating-webhook-configuration
+    app.kubernetes.io/instance: 'seldon-core'
+    app.kubernetes.io/name: 'seldon-core-operator'
+    app.kubernetes.io/version: '1.6.0'
+  name: seldon-validating-webhook-configuration-kubeflow
 webhooks:
 - clientConfig:
     caBundle: Cg==
     service:
       name: seldon-webhook-service
-      port: {{ webhook_port }}
```

Comment:
I think we'll still need this line and the other similar ones in this file, as IIRC the default is `443`, and Seldon defaults to `4443`.

Reply:
+1, updating now.

Reply:
Done, resolving this